include/nng/protocol/mqtt/mqtt_parser.h

```diff
@@ -11,11 +11,11 @@
 // Do not change to %lu! just supress the warning of compiler!
 #define DISCONNECT_MSG          \
 	"{\"username\":\"%s\"," \
-	"\"ts\":%llu,\"reason_code\":\"%x\",\"client_id\":\"%s\"}"
+	"\"ts\":%lu,\"reason_code\":\"%x\",\"client_id\":\"%s\"}"
 
 #define CONNECT_MSG                                                           \
 	"{\"username\":\"%s\", "                                              \
-	"\"ts\":%llu,\"proto_name\":\"%s\",\"keepalive\":%d,\"return_code\":" \
```

Comment:
I made this deliberately, intend to be compatible with ARM32

Reply:
please remove this change

Reply:
That's change removed.

Reply:
I prefer using `cJSON` to format json string.